src/runtime/middleware/i18n-default-locale.ts

```diff
@@ -0,0 +1,135 @@
+import {
```

Comment:
this whole file is drastically lacking comments.

What's the overall purpose of this file
What are the specific management for each method in this file